php/libraries/Module.class.inc

```diff
@@ -85,6 +85,7 @@ class Module extends \LORIS\Router\PrefixRouter implements RequestHandlerInterfa
                     $fpath = $mpath . "/php/"
                     . strtolower(substr($class, strlen("LORIS\\$name\\")))
                     . ".class.inc";
+                    $fpath = str_replace('\\', '/', $fpath);
```

Comment:
Why?

Reply:
@johnsaigle that's a fix that was pushed into minor and needed for this PR to work with the endpoints directory in the genomic_browser. Otherwise the wrong slashes are used and class isn't found.

Reply:
Cool, I didn't know. 👍